src/inline.d

```diff
@@ -35,8 +35,10 @@ import ddmd.tokens;
 import ddmd.visitor;
 
 private:
-enum LOG = false;
-enum CANINLINE_LOG = false;
+
+enum LOG = 0;
+enum CANINLINE_LOG = 0;
+enum EXPANDINLINE_LOG = 0;
```

Comment:
The trouble with using `0` is it implies that numbers other than `0` or `1` are possible, and so I spent time fruitlessly looking for that possibility. I recommend sticking with `true` and `false`.